.github/workflows/package-manager-python-demo.yml

```diff
@@ -0,0 +1,46 @@
+name: python-package-manager-demo
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        r-version: ['3.6.3', '4.1.3']
```

Comment:
Remove r-version?

Reply:
Oh, good catch. I forgot to clean these up when I switched to two separate workflows.